scenario-api/src/main/scala/pl/touk/nussknacker/engine/canonicalgraph/CanonicalProcess.scala

```diff
@@ -78,12 +78,12 @@ object CanonicalProcess {
 
 //in fact with branches/join this form is not canonical anymore - graph can be represented in more than way
 case class CanonicalProcess(
-    metaData: MetaData,
-    // separation of nodes and additionalBranches is just for compatibility of stored json
-    // DON'T use these fields, rely on allStartNodes or mapAllNodes instead.
-    nodes: List[CanonicalNode],
-    additionalBranches: List[List[CanonicalNode]] = List.empty,
-    stickyNotes: List[StickyNote] = List.empty,
+                             metaData: MetaData,
```

Comment:
Could you reformat the changes? It looks like the indentation doesn't match the configured scalafmt.